cmd/agent/main.go

```diff
@@ -485,3 +483,18 @@ func reconcileKubelet(ctx context.Context) (bool, error) {
 	}
 	return false, nil
 }
+
+func setLogrus() {
+	logrus.SetOutput(colorable.NewColorableStdout())
+
+	if os.Getenv("CATTLE_TRACE") == "true" || os.Getenv("RANCHER_TRACE") == "true" {
+		logrus.SetLevel(logrus.TraceLevel)
+		return
```

Comment:
Somewhat of a nit/opinion thing, but prefer not to use `return` here as if we're to add more logic to setting up logrus further in the function (starting at line 498 or below) it may be hard to notice it will be skipped if trace logging is set.

So as much I understand the desire not to increase nesting level something like below will be safer:
```golang
	if os.Getenv("CATTLE_TRACE") == "true" || os.Getenv("RANCHER_TRACE") == "true" {
		logrus.SetLevel(logrus.TraceLevel)
	} else if os.Getenv("CATTLE_DEBUG") == "true" || os.Getenv("RANCHER_DEBUG") == "true" {
		logrus.SetLevel(logrus.DebugLevel)
	}
```

Reply:
Done. 
I'll try to think on that for the next PRs.